experimental/MorphismOfComplexes/test/lift_resolution.jl

```diff
@@ -0,0 +1,30 @@
+@testset "lifting morphisms through free resolutions" begin
+  R, (x, y, z) = QQ[:x, :y, :z]
+
+  I = ideal(R, [x, y, z])
+  J = ideal(R, [x^2, y^2, z^2])
+
+  F = FreeMod(R, 1)
+  IF, _ = I*F
+  JF, _ = J*F
+
+  M1, _ = quo(F, IF)
+  M2, _ = quo(F, JF)
+
+  phi = hom(M2, M1, gens(M1))
+
+  res1 = free_resolution(M1).C
+  res2 = free_resolution(M2).C
+
+  psi = Oscar.lift_morphism_through_free_resolutions(phi, domain_resolution=res2, codomain_resolution=res1);
```

Comment:
This is the command to lift a morphism of modules `phi : M -> N` to a morphism of their resolutions.